Modix.Services/Moderation/ModerationService.cs

```diff
@@ -246,7 +246,17 @@ public async Task CreateInfractionAsync(InfractionType type, ulong subjectId, st
             AuthorizationService.RequireClaims(_createInfractionClaimsByType[type]);
 
             var guild = await DiscordClient.GetGuildAsync(AuthorizationService.CurrentGuildId.Value);
-            var subject = await UserService.GetGuildUserAsync(guild.Id, subjectId);
+
+            IGuildUser subject;
+            try
```

Comment:
Booooooo for exception-based flow-control. Is there no way to check whether a user exists, before attempting to Get() it?

Reply:
100% agreed - but, not without refactoring `UserService`, unfortunately. I'll look into that, maybe a `TryGetUser` or similar.